### DISCUSSION TITLE
show getStatesFor is ToggleButton in Filament PHP 3

### DISCUSSION THREAD

**Discussion by Grazulex** [spatie/laravel-model-states#263] [Q&A]
Hello,

have somebody a idee how I can use this package with Filament PHP and more on a form with ToggleButton with color and icons ?
Thanks

**Comment by A909M** [comment]
if states aren't transitioned using the Transition class and are directly saved to the database like regular fields.
you can implement  the HasLabel and HasIcon interface.
https://filamentphp.com/docs/3.x/forms/fields/toggle-buttons

**Comment by A909M** [comment]
if states aren't transitioned using the Transition class and are directly saved to the database like regular fields.
you can implement  the HasLabel and HasIcon interface.
https://filamentphp.com/docs/3.x/forms/fields/toggle-buttons
